util/logtools/make_timeline.ts

```diff
@@ -357,7 +357,7 @@ const assembleTimelineStrings = (
   let lastAbilityTime = (start.getTime());
   let timelinePosition = 0;
   let lastEntry: TimelineEntry = { time: lastAbilityTime.toString(), lineType: 'None' };
-  if (fight.sealName) {
+  if (fight.sealName !== undefined && fight.sealName.length > 0) {
```

Comment:
```suggestion
  if (fight.sealName !== undefined) {
```

`sealName` is already checked for empty in encounter tools.  If set, then it's either valid or undefined.